src/components/FullHierarchyTable/FullHierarchyTable.tsx

```diff
@@ -249,7 +249,7 @@ export const FullHierarchyTable: React.FunctionComponent<FullHierarchyTableAllPr
                     }
                 }))
         ]
-    }, [depthLevel, fields, props.meta.name, textFilters])
+    }, [depthLevel, fields, props.meta.name, textFilters, props.rowMetaFields])
```

Comment:
Memoization seems wrong on that one.
Aside of `props.rowMetaFields`, `maxDepth` is also used but not specified although it's obviously changes with data; `indentColumn` is also used but not specified and will need memoization of its own